dom-event-delegation/dom-event-delegation-quiz-notes.md

```diff
@@ -0,0 +1,53 @@
+# dom-event-delegation-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the `event.target`?
+The event we are listening for in the addEventListener() method.
```

Comment:
the `event.target` would be the **element** on the page where the event took place.